src/components/background-video.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
```

Comment:
This is probably a dumb question, but do we really want this to be purely client? If someone's using this for a big part of their web page, we probably still want the `children` to get SSRd, right? The player itself being client-only feels obvious, but is it necessary to do it for the entire component?

Reply:
Good question, client components are still SSR'd and hydrated by default. you can see if you look at the string html source, all the tags are there video, img, blur-up thumb inlined...
https://github.com/reactwg/server-components/discussions/4